css/index.css

```diff
@@ -1,3 +1,278 @@
+    }
+
+    .logo {
+        display: flex;
+        width: 100%;
+        order: 1;
+        justify-content: center;
+        margin: 30px 0;
+    }
+}
+
+@media(max-width: 500px) {
+    .flex-nav {
+        flex-direction: column;
+    }
+}
+
+.flex-nav a {
+	color: grey;
+	text-decoration: none;
+}
+
+@media(max-width: 800px) {
+    
+    nav a {
+        order: 2;
+        justify-content: space-around;
+    }
+}
+
+@media(max-width: 500px) {
+    @keyframes rainbowText {
+        0% {background-color: yellow;}
+        25% {background-color: green;}
+        50% {background-color: blue;}
+        75% {background-color: red;}
+    }
+    nav a {
+        color: black;
+        display: flex;
+        justify-content: space-around;
+        align-items: center;
+        width: 100%;
+        border-top: 1px solid grey;
+        height: 30px;
+        animation-name: rainbowText;
+        animation-duration: 10s;
+        animation-iteration-count: infinite;
+        animation-direction: alternate;
+        }
+    
+    
+    nav a:nth-child(6) {
+        border-bottom: 1px solid grey;
+    }
+}
+
+.header-left {
+	margin-top: 50px;
+	display: inline-block;
+	width: 49%;
+	vertical-align: middle;
+	text-align: center;	
+}
+
+@media(max-width: 800px) {
+    .header-left {
+        width: 100%;
+        margin-bottom: 30px;
+    }
+}
+
+.header-right {
+	margin-top: 50px;
+	vertical-align: middle;
+	text-align: center;
+	display: inline-block;
+	width: 49%;
+}
+
+@media(max-width: 800px) {
+    .header-right {
+        display: none;
+    }
+}
+
+header h1 {
+	font-size: 80px;
+}
+
+@media(max-width: 500px) {
+    header h1 {
+        font-size: 45px;
+    }
+}
+
+header button {
+	height: 30px;
+	width: 150px;
+	border: 1px solid black;
+}
+
+.top-content {
+	margin-bottom: 20px;
+	margin-top: 25px;
+	display: inline-block;
+	width: 100%;
+}
+
+.top-content-right {
+	text-align: left;
+	display: inline-block;
+	width: 49%;
+}
+
+@media(max-width: 500px) {
+    .top-content-right {
+        width: 100%;
+        padding-left: 10px;
+        margin-top: 20px;
+        margin-bottom: 10px;
+    }
+}
+
+.top-content-left {
+	text-align: left;
+	display: inline-block;
+	width: 49%;
+}
+
+@media(max-width: 500px) {
+    .top-content-left {
+        width: 100%;
+        padding-left: 10px;
+    }
+}
+
+.middle-img {
+	width: 100%;
+	margin: auto;
+}
+
+.bottom-content {
+	margin-top: 20px;
+	margin-bottom: 25px;
+	width: 100%;
+	display: inline-block;
+}
+
+.bottom-content div {
+	display: inline-block;
+	width: 32%;
+}
+@media(max-width: 800px) {
+    .bottom-content div {
+        padding-left: 10px; 
+    }
+}
+
+@media(max-width: 500px) {
+    .bottom-content div {
+        width: 100%;
+        padding-left: 10px;
+        margin-bottom: 10px;
+        margin-top: 10px;
+    }
+}
+
+.copy-right {
+	margin-top: 50px;
+	text-align: center;
+}
+
+@media(max-width: 800px) {
+    .contact {
+        padding-left: 10px;
+    }
+}
+
```

Comment:
A good practice is to place your media queries directly below each element for organizing.  Another approach is to do a giant media query at the bottom and nest all the selectors at once.  Dan Levy's recommendation is to do the former.